### ISSUE TITLE
FAQ: SPAdes, and thus Jovian, is only comparable across systems/installations if the same amount of threads are used for assembly.

### ISSUE THREAD

**Issue by DennisSchmitz** [DennisSchmitz/Jovian_archive#145]
**TLDR** *The `Jovian` output is only comparable between systems/installations with 12 or more computing threads.*

After running a demo dataset, with known outputs/hashes, on a system with limited resources (<12 computing threads) the hashes of the output files didn't match the expected values. I traced this back to the `SPAdes` output. After some digging, I found that `SPAdes` output differs based on the number of threads that are used. On this system, `snakemake` lowered the number of threads (default=12) for the de novo assembly rule to abide the available resources, thus generating different output, and deviant hashes.

I'll keep this issue open as a reminder to avoid future confusion and hopefully this saves someone some time.
